spanner/api/Spanner.Samples/WriteUsingDmlCoreAsync.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_dml_getting_started_insert]
+
+using Google.Cloud.Spanner.Data;
+using System;
+using System.Threading.Tasks;
+
+public class WriteUsingDmlCoreAsyncSample
+{
+    public async Task<int> WriteUsingDmlCoreAsync(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        using var connection = new SpannerConnection(connectionString);
+        await connection.OpenAsync();
+
+        SpannerCommand cmd = connection.CreateDmlCommand(
+            "INSERT Singers (SingerId, FirstName, LastName) VALUES "
+               + "(17, 'Melissa', 'Garcia'), "
+               + "(18, 'Russell', 'Morales'), "
+               + "(19, 'Jacqueline', 'Long')");
+        int rowCount = await cmd.ExecuteNonQueryAsync();
```

Comment:
This doesn't quite work because there are specific inputs and outputs in the getting started guide. At the very least, there should be printing out "4 row(s) inserted..." according to the [Docs](https://cloud.google.com/spanner/docs/getting-started/csharp#write-data).

If you insert different data, the getting started guide may not make sense. I haven't tried it from start to end to know what would happen.